src/Tribe/Views/V2/Template.php

```diff
@@ -59,8 +59,12 @@ class Template extends Base_Template {
 	public function render( array $context_overrides = [] ) {
 		$context = wp_parse_args( $context_overrides, $this->context );
 		$context['_context'] = $context;
+		$template_slug = array_filter( [
+			$this->view->get_template_path(),
+			$this->view->get_template_slug(),
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: d7d913f371b8750842db72ed46f5861a</sub>